### DISCUSSION TITLE
Chia 2.0 Windows 11 bladebit_cuda

### DISCUSSION THREAD

**Discussion by mbegunok** [Chia-Network/chia-blockchain#16171] [Q&A]
Sorry for asking simple question.
Can't find bladebit_cuda in any windows directiry. Is that possible to use CLI in windows?

**Comment by BrandtH22** [comment]
Yes, however there is currently a bug in the windows build for bladebit that leads users to have to manually setup the binary rather than using the packaged installer for windows. This bug should be fixed during the next release but the bladebit source code page is here if interested: https://github.com/Chia-Network/bladebit/releases/tag/v3.0.0

Side note - you can generally find more information and more immediate support on these types of issues in our discord server: https://discord.gg/chia
